docker-compose.test.yml

```diff
@@ -0,0 +1,12 @@
+version: "2.4"
+
+services:
+  sut:
+    build: .
+    image: localhost:5000/sut
+    privileged: true
```

Comment:
Is privileged actually required?

Reply:
Perhaps not anymore, but some permissions will be needed for mounting.